src/Tabs/__tests__/Tabs.test.tsx

```diff
@@ -7,7 +7,7 @@ import { ENTER_KEY, SPACEBAR } from '../../constants';
 
 const render = (props = {}, tabProps = {}) =>
   shallow(
-    <Tabs {...props}>
+    <Tabs width="100%" {...props}>
```

Comment:
Added `width` here to enforce type-support for pass-through props to `StyledTabs`.